features/git-town/setup/install_fish_autocomplete.feature

```diff
@@ -7,34 +7,27 @@ Feature: Installing Fish Shell autocomplete definitions
 
   Scenario: without existing fish autocompletion folder
     Given I have no fish autocompletion folder
-    When I run `git town install-fish-autocompletion`
-    Then it runs the commands
-      | COMMAND                                                                                   |
-      | mkdir -p ~/.config/fish/completions                                                       |
-      | ln -s <%= GIT_TOWN_DIRECTORY %>/autocomplete/git.fish ~/.config/fish/completions/git.fish |
+    When I run `gt install-fish-autocompletion`
+    Then I see "Git autocompletion for Fish shell installed"
+    And I have a Git autocompletion file
 
 
   Scenario: with empty fish autocompletion folder
     Given I have an empty fish autocompletion folder
-    When I run `git town install-fish-autocompletion`
-    Then it runs the commands
-      | COMMAND                                                                                   |
-      | ln -s <%= GIT_TOWN_DIRECTORY %>/autocomplete/git.fish ~/.config/fish/completions/git.fish |
+    When I run `gt install-fish-autocompletion`
+    Then I see "Git autocompletion for Fish shell installed"
+    And I have a Git autocompletion file
```

Comment:
Going to merge as is since existing steps use this wordage. We can update it, in a separate PR if you'd like.